devices/sunricher.js

```diff
@@ -25,6 +25,18 @@ module.exports = [
         toZigbee: [],
         meta: {multiEndpoint: true},
     },
+    {
+        zigbeeModel: ['ZGRC-KEY-009'],
+        model: 'YPHIX 50208693',
+        vendor: 'Sunricher',
+        description: 'Zigbee Wall Remote Control for RGBW, 1 Zone with 2 Scenes',
+        fromZigbee: [fz.command_on, fz.command_off, fz.command_move, fz.command_stop, fz.battery,
+            fz.command_recall, fz.command_step, fz.command_move_to_color, fz.command_move_to_color_temp],
+        toZigbee: [],
+        exposes: [e.battery(), e.action(['on', 'off',
+            'brightness_move_up', 'brightness_move_down', 'brightness_move_stop', 'brightness_step_up', 'brightness_step_down',
+            'recall_*'])],
```

Comment:
Instead of using the wildcard `recall_*` here, please add all the send value (e.g. `recall_1`, etc..)

Reply:
Done :)